src/client/app/components/HeaderButtonsComponent.tsx

```diff
@@ -77,15 +76,16 @@ export default function HeaderButtonsComponent() {
 	// Local state for rendering.
 	const [state, setState] = useState(defaultState);
 	// Information on the current user.
-	const currentUser = useSelector((state: State) => state.currentUser.profile);
+	const { profile: currentUser } = useAppSelector(selectCurrentUser);
 	// Tracks unsaved changes.
-	const unsavedChangesState = useSelector((state: State) => state.unsavedWarning.hasUnsavedChanges);
+	// TODO Re-implement AFTER RTK Migration
```

Comment:
Is unsaved still used and status of this comment? Does it need to be done later per TODO below?

Reply:
Unsaved warning needs an overall update and is planned to get re-implemented post-rtk migration.